app/Http/Controllers/ProductController.php

```diff
@@ -13,9 +14,8 @@ class ProductController extends Controller
      */
     public function index(Request $request)
     {
-        return view('product.index')->with([
-            'token' => $request->cookie('access_token'),
-            'categories' => json_encode(ProductCategory::getOptions())
+        return Inertia::render('Product/ListComponent', [
+            'categoryEnum' => ProductCategory::getOptions()
```

Comment:
In almost all ecommerce systems we've handled, it's common that Product Categories are editable by the client's company. A model for ProductCategory would have been better.